src/Athena/Views/Home/Home.cshtml

```diff
@@ -5,14 +5,34 @@
 
 @if (!User.Identity.IsAuthenticated)
 {
-    <div class="notLoggedIn" style="padding-top:5em">
-        <h1>
-            Welcome to the Athena Scheduling Assistant!
-        </h1>
-        <p>
-            This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
-        </p>
+<div class="notLoggedIn" style="padding-top:5em">
+    <h1>
+        Welcome to the Athena Scheduling Assistant!
+    </h1>
+    <p>
+        This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
+    </p>
+    <div class="row center">
+        <div class="col s4">
+            <i class="large material-icons">person_add</i>
+
+            <h4>Security</h4>
+            <p>Uses your Google or Microsoft account for easy authentication. We provide a means of keeping your information safe by using the same authentication services as most universities.</p>
+        </div>
+        <div class="col s4">
+            <i class="large material-icons">event_available</i>
```

Comment:
Lets color the icons by adding these classes: `blue-text text-accent-3`

Reply:
thats fixed